modules/eventschedule.php

```diff
@@ -167,7 +167,9 @@ function parseWorkingHours($hours_period)
 
 $userid = $filter['userid'];
 
-$userlist = $LMS->GetUserNames();
+$params['access'] = 1;
```

Comment:
Coś mi się wydaje, że takie ustawienie $params['access'] na sztywno jeden spowoduje pomnięcie tej właściwości z $filter.

Reply:
Lista użytkowników w selekcie nie pokazuje użytkowników bez dostępu. Natomiast w pętli niżej dane są brane z innego miejsca. Mogę zrobić po użyciu funkcji unset($params['access']); ale czy to potrzebne?

Reply:
Zrobiłem test - wprowadzona zmiana nie psuje filtra "Użytkownicy".

Reply:
Coś się zrypało. Po git push mam status tego PR jako merged a PR ma status closed.
Wrzuciłem nowy PR.